.github/workflows/pull-request.yml

```
@@ -107,13 +122,26 @@
         uses: actions/setup-python@v3
         with:
           python-version: 3.11
-      - name: Install dependencies
+      - name: Install Ubuntu packages
+        run: sudo apt-get install -y python3 python3-pip snmp libsnmp-dev snmp-mibs-downloader gcc python-dev-is-python3 python3-venv
+      - name: Cache pip packages
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)